jobs/database-seeder/spec

```diff
@@ -0,0 +1,46 @@
+---
+name: database-seeder
+description: >
+  When using an external database server, seed it with the necessary databases.
+  This is done during pre-start. The run operation is present only to satisfy
+  parts of the system expecting an actual task. Which does nothing, except to
+  exist, to be called.
```

Comment:
We only need the first line in the description here; the other lines are more appropriate inside the run script, I think.

Reply:
Will change.

Reply:
Done with commit bb4abc33c3eb8da3ecbf36bb79f791fd63ecf620